services/indexeddbService.ts

```diff
@@ -102,10 +102,14 @@ export async function updateSiteFeed(siteFeed: SiteFeed) {
     await update(db, SiteFeedTable.Name, siteFeed);
 
     db.close();
+
+    return;
```

Comment:
Can you explain why are there returns needed?😊

Reply:
Sure! Since the code is being executed in a callback I'd thought I'd add a return statement so we can await it properly